### PR TITLE
Issue 5855: Sweep over of KeyValueTable related tasks to a new Controller host

### LINKED ISSUE #5855: On Controller failover,  KeyValueTable related tasks do not get moved over to a new Controller host.
**Describe the bug**
When a Controller server goes down, another Controller tries to take over its tasks by "sweeping" the tasks left incomplete by the earlier Controller. But for KeyValueTable workflows this sweepover to new Controller process does not happen

**Additional context**
We do not have code for handling `kvtRequestEventProcesser` in the `sweepFailedProcesses` method:
https://github.com/pravega/pravega/blob/master/controller/src/main/java/io/pravega/controller/server/eventProcessor/ControllerEventProcessors.java#L210

### PR DESCRIPTION
**Pull request by kotlasaicharanreddy**
**Change log description**  
- When a Controller server goes down, another Controller tries to take over its tasks by "sweeping" the tasks left incomplete by the earlier Controller.
- `sweepFailedProcesses()` and `handleFailedProcess()` methods handles the takeover of incomplete tasks left.
- This PR addresses the issue of missing code for `KeyValueTable` workflows in the above methods.

**Purpose of the change**  
Fixes #5855

**What the code does**  
- Handles the `KeyValueTable` workflow of sweeping the tasks left incomplete by a failed controller in `sweepFailedProcesses()` and `handleFailedProcess()` methods

**How to verify it**  
- `testHandleOrphaned()` test should always pass.
- Build should succeed and all test must pass.

### CODE REVIEW

**Review by shiveshr — COMMENTED** [review]
can we add unit tests and mark this PR as ready

**Comment by kotlasaicharanreddy** [comment]
> can we add unit tests and mark this PR as ready

Added tests and code coverage is green.

File: controller/src/test/java/io/pravega/controller/server/eventProcessor/ControllerEventProcessorsTest.java [hunk 1]
```diff
@@ -98,9 +99,11 @@ public void testHandleOrphaned() {
...
         EventProcessorSystem system = mock(EventProcessorSystem.class);
         EventProcessorGroup<ControllerEvent> processor = getProcessor();
+        EventProcessorGroup<ControllerEvent> mockProcessor = spy(processor);
 
+        doThrow(new CheckpointStoreException("host not found")).when(mockProcessor).notifyProcessFailure("host3");
         try {
-            when(system.createEventProcessorGroup(any(), any(), any())).thenReturn(processor);
+            when(system.createEventProcessorGroup(any(), any(), any())).thenReturn(mockProcessor);
         } catch (CheckpointStoreException e) {
             e.printStackTrace();
```
Comment (shrids) [inline controller/src/test/java/io/pravega/controller/server/eventProcessor/ControllerEventProcessorsTest.java:108]: remove this catch block.

Comment (kotlasaicharanreddy) [reply controller/src/test/java/io/pravega/controller/server/eventProcessor/ControllerEventProcessorsTest.java]: Thanks for reviewing, Removed catch block.

**Review by shrids — APPROVED** [review]

**Review by shiveshr — APPROVED** [review]

**Pull request merged**